src/onegov/org/views/editor.py

```diff
@@ -74,6 +80,36 @@ def handle_new_page(self, request, form, src=None, layout=None):
     }
 
 
+def handle_new_root_page(self, request, form, src=None, layout=None):
```

Comment:
I'm guessing the `src` parameter was created as a result of copy-pasting from `handle_new_page`. 
It's not needed and  always == `None`, it can be removed.

Reply:
Or do wee need to be able to paste to the root as well?

Reply:
I would not support 'paste' on root level. So let me remove the 'src' param from handle_new_root_page

Reply:
done